sql/src/main/java/io/crate/jobs/ESJobContext.java

```diff
@@ -101,6 +101,11 @@ public boolean isKilled() {
         return iskilled;
     }
 
+    @Override
+    public PageDownstreamContext pageDownstreamContext(int inputId) {
+        return null;
```

Comment:
would it make sense to use byte here as type in order to more match the range  of the var